src/main/java/net/corevalue/multitenant/poc/config/OrientDBConfiguration.java

```diff
@@ -22,7 +22,8 @@
 
     @Bean
     OrientDB orientDB(MultiTenantProperties props) {
-        return new OrientDB(String.format("remote:%s", props.getOrientDb().getHost()), OrientDBConfig.defaultConfig());
```

Comment:
with ConfigurationProperties use @Value

Reply:
What is the advantage of using the @value approach?